### PR TITLE
Amendments

### PR DESCRIPTION
**Pull request by janvanmansum**
- [x] Review texts
- [x] Review assignments

### CODE REVIEW

**Review by janvanmansum — COMMENTED** [review]
- I think that when talking about events, usually the events are called `next`, `completed`, `error`, etc and the _event handlers_ are called `onNext`, `onCompleted`, `onError` (i.e. they specify what happens "on receiving the xxx event").

**Comment by rvanheest** [comment]
I see what you're saying, and the reasoning is completely correct. However, they are actually called `onNext` (etc.) events.

File: src/main/scala/workshop4/01 Iterable.sc [hunk 1]
```diff
@@ -28,3 +28,5 @@ def generateInfiniteCollection: Iterable[Double] = {
     }
   }
 }
+
```
Comment (rvanheest) [inline src/main/scala/workshop4/01 Iterable.sc]: why these newlines?

Comment (janvanmansum) [reply src/main/scala/workshop4/01 Iterable.sc]: I'll remove those. I obviously was playing with the worksheet and left those newlines there by accident. I am still busy reviewing by the way, so hang on for updates to this PR.

Comment (rvanheest) [reply src/main/scala/workshop4/01 Iterable.sc]: I will! And good to see that you're trying out stuff!

**Comment by janvanmansum** [comment]
Okay, if that is customary in the Rx-world, so be it. It seems a bit odd though, to convolute the event and the event handler.

**Comment by rvanheest** [comment]
Thank for all the work you put into improving this PR. I saw you added a couple of TODO's as well. I'll resolve those ASAP.

@jo-pol given your current work on EASY, I assume you have not yet started the review. Could you wait with that until I finished resolving the feedback from this PR? I'll let you know when everything's finished!

**Pull request merged**